### PR TITLE
Allow adding a * to the alias list to rewrite all emails that are not matched by the other rules

### PR DESCRIPTION
**Pull request by TimoKoole**
Fixes 
In our setup on our Reproduction environment we have actual emails of customers, however we never want these to go out during testing. Therefore we want to alias all emails to our test mailbox.

Changes proposed in this pull request:
- Allow adding a * to the alias list to rewrite all emails that are not matched by the other rules

Checklist:
- [x] docs updated
- [x] tests updated
- [x] Changes.md updated
- [x] package.json.version bumped
- [ ] published to NPM (will be done by @core)

### CODE REVIEW

File: Changes.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 # 1.0.0 - 2017-09-01
 
 - imported from haraka
+
+# 1.0.1 - 2017-09-01
```
Comment (msimerson) [inline Changes.md]: Is that date right?

**Comment by msimerson** [comment]
You are effectively creating an email catchall. These were quite popular for a time, but then became considered by many to be a not-very-good idea because spammers just invented random email address at your domain and dumped on them, in a big way, until you relented and killed off the catchall.

File: Changes.md (hunk 1 above)
Comment (TimoKoole) [reply Changes.md]: It is not, I fixed it to 2020-09-02

**Comment by TimoKoole** [comment]
> You are effectively creating an email catchall. These were quite popular for a time, but then became considered by many to be a not-very-good idea because spammers just invented random email address at your domain and dumped on them, in a big way, until you relented and killed off the catchall.

The thing is that our use case is rather unusual I guess? The Haraka server will only allow incoming traffic from within the corporate network, and we want to catch all mail going out. The alternative in current setup would be to create a alias for every existing email domain in our customer database and constantly keep that up to date, which does scale well at all.

**Comment by msimerson** [comment]
> The thing is that our use case is rather unusual I guess?

Maybe, but Haraka is a `running with scissors` MTA, so we should not stand in the way. Sorry for not merging this sooner.

**Pull request merged**